FlowCryptCommon/Extensions/Data/DataExntensions+ZBase32Encoding.swift

```diff
@@ -0,0 +1,158 @@
+//
+//  DataExntensions+Encoding.swift
+//  FlowCryptCommon
+//
+//  Created by Yevhen Kyivskyi on 17.05.2021.
+//  Copyright © 2021 FlowCrypt Limited. All rights reserved.
+//
+
+import Foundation
+
+public extension Data {
```

Comment:
Is this a re-implementation or a copy-paste of open-source code?

If mostly copy-paste, original authors and their license should be credited.

Reply:
It's re-implementation. I used a part of Base32 encoding and adopt it to zBase32.